src/components/IOUConfirmationList.js

```diff
@@ -200,17 +162,26 @@ class IOUConfirmationList extends Component {
 
     /**
      * Returns the sections needed for the OptionsSelector
-     * @param {Boolean} maxParticipantsReached
+     *
+     * @param {Array} participants
      * @returns {Array}
      */
-    getSections() {
+    getSections(participants) {
         const sections = [];
         if (this.props.hasMultipleParticipants) {
-            const selectedParticipants = this.getSelectedParticipants();
-            const unselectedParticipants = this.getUnselectedParticipants();
-
+            const [selected, unselected] = _.reduce(participants, (memo, participant) => {
+                if (participant.selected) {
+                    memo[0].push(participant);
+                } else {
+                    memo[1].push(participant);
+                }
+                return memo;
+            }, [[], []]);
+            const selectedParticipants = selected || [];
+            const unselectedParticipants = unselected || [];
```

Comment:
NAB the `|| []` is not necessary there is no way this won't be at least an empty array.